src/BuiltInTools/BrowserRefresh/WebSocketScriptInjection.js

```diff
@@ -101,7 +101,8 @@ setTimeout(async function () {
       applyUpdateCapabilities = window.Blazor._internal.getApplyUpdateCapabilities();
     } catch (error) {
       console.warn(error);
-      applyUpdateCapabilities = sendErrorToClient ? '!' + error.stack : '';
+
+      applyUpdateCapabilities = sendErrorToClient ? (error.stack || error.message || '<unknown error>') : '';
```

Comment:
Could we include `error.message` even if `error.stack` is truthy? Firefox, unlike Chromium, doesn't include the error message itself in `Error.stack`.